packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -2090,7 +2094,7 @@ class Route extends EmberObject.extend(ActionHandler, Evented) implements IRoute
     let controllerName = this.controllerName || this.routeName;
     let owner = getOwner(this);
     let controller = owner.lookup<Controller>(`controller:${controllerName}`);
-    let queryParameterConfiguraton = get(this, 'queryParams');
+    let queryParameterConfiguraton = get(this, 'queryParams') as this['queryParams'];
```

Comment:
Huh. Is this a knock-on effect from having `get` not be well-typed? If so, maybe we right types which understand how to do key lookup for it for internal purposes, at least?